README.md

```diff
@@ -1,18 +1,17 @@
+[![Published on NPM](https://img.shields.io/npm/v/@polymer/iron-iconset-svg.svg)](https://www.npmjs.com/package/@polymer/iron-iconset-svg)
 [![Build status](https://travis-ci.org/PolymerElements/iron-iconset-svg.svg?branch=master)](https://travis-ci.org/PolymerElements/iron-iconset-svg)
-
-_[Demo and API docs](https://elements.polymer-project.org/elements/iron-iconset-svg)_
-
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://webcomponents.org/element/@polymer/iron-iconset-svg)
 
 ## &lt;iron-iconset-svg&gt;
 
-The `iron-iconset-svg` element allows users to define their own icon sets
-that contain svg icons. The svg icon elements should be children of the
+The `iron-iconset-svg` element allows users to define their own icon sets that
+contain svg icons. The svg icon elements should be children of the
 `iron-iconset-svg` element. Multiple icons should be given distinct id's.
 
-Using svg elements to create icons has a few advantages over traditional
-bitmap graphics like jpg or png. Icons that use svg are vector based so
-they are resolution independent and should look good on any device. They
-are stylable via css. Icons can be themed, colorized, and even animated.
+Using svg elements to create icons has a few advantages over traditional bitmap
+graphics like jpg or png. Icons that use svg are vector based so they are
+resolution independent and should look good on any device. They are stylable
+via css. Icons can be themed, colorized, and even animated.
 
 Example:
```

Comment:
These are just copy pasted from the element docs, no? I think we're just linking to them, and deleting everything else, as per https://raw.githubusercontent.com/PolymerElements/paper-input/85c6cfd45ba0f0c6c76bceee93b5f18890e2c655/README.md

Reply:
SGTM; I replaced this top section with the links in the example there.